tests/unit/dataactvalidator/test_a34_appropriations.py

```diff
@@ -0,0 +1,33 @@
+from tests.unit.dataactcore.factories.domain import SF133Factory
+from tests.unit.dataactcore.factories.staging import AppropriationFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+
+
+_FILE = 'a34_appropriations'
+_TAS = 'a34_appropriations_tas'
+
+
+def test_column_headers(database):
+    expected_subset = {'row_number', 'budget_authority_unobligat_fyb', 'sf_133_amount'}
+    actual = set(query_columns(_FILE, database))
+    assert (actual & expected_subset) == expected_subset
+
```

Comment:
I'm not too sure that's necessary. It's just going to return 0 errors since the fiscal year/tas won't match so it wouldn't do much benefit, unless there is some inherent benefit which i'm not seeing (please correct me if i'm wrong, i'm not too sure about this).